common/helpers/date.test.js

```diff
@@ -3,6 +3,9 @@ const {
   getCurrentDayAsRange,
   getRelativeDate,
   getDateRange,
+  isCurrentWeek,
```

Comment:
There is a `_isCurrentWeek` function in `nunjucks/filters` but it is "private" and untested, and I also need PreviousWeek and NextWeek